src/main/java/at/fh/joanneum/irfc/service/EventCategoryService.java

```diff
@@ -0,0 +1,84 @@
+package at.fh.joanneum.irfc.service;
+
+import at.fh.joanneum.irfc.model.event.EventCategoryDTO;
+import at.fh.joanneum.irfc.model.event.EventCategoryMapper;
+import at.fh.joanneum.irfc.persistence.entiy.EventCategoryEntity;
+import at.fh.joanneum.irfc.persistence.repository.EventCategoryRepository;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+
+/**
+ * @author dominik.jaeger@edu.fh-joanneum.at
+ **/
+@RequestScoped
+public class EventCategoryService {
+
+  @Inject
+  EventCategoryRepository eventCategoryRepository;
+  public List<EventCategoryDTO> getAll() {
+    return eventCategoryRepository.listAll().stream()
+        .map(EventCategoryMapper.INSTANCE::toDto)
+        .collect(Collectors.toUnmodifiableList());
+  }
+
+  public EventCategoryDTO get(Long id) {
```

Comment:
Warum verwendest du hier den Long Wrapper?

Reply:
cause the id is a Long in our defined data model